mkdocs.yml

```
@@ -75,4 +75,6 @@ nav:
   - 'Guides':
     - 'Managing packages': guides/managing-packages.md
     - 'Managing layers': guides/managing-layers.md
+  - 'Development Process':
+      - 'Standard Release Process': dev-proc/standard-release-process.md
```

```suggestion
      - 'Standard Release Process': development-process/standard-release-process.md
```

And change directory name to `development-process`. That way paths will be consistent for each menu (lower case, space changed to `-`)